ios/RNGestureHandlerButton.m

```diff
@@ -45,5 +45,11 @@ - (UIView *)hitTest:(CGPoint)point withEvent:(UIEvent *)event
     return inner;
 }
 
+- (void)layoutSubviews
+{
+    self.clipsToBounds = YES;
```

Comment:
Can you explain what this change is for? Unless there is some weird set up it is not the best idea to do it in layoutSubviews, would be much better to just do it one in init or somewhere

Reply:
If you do set some value of radius button on iOS, u will notice that highlight does not clip to bounds. It does change it